cluster/charts/rook-ceph-cluster/templates/ingress.yaml

```diff
@@ -24,11 +24,11 @@ spec:
               service:
                 name: rook-ceph-mgr-dashboard
                 port:
-                  name: http-dashboard
+                  name: https-dashboard
```

Comment:
Whether it is the `http-dashboard` or `https-dashboard` port will depend on if the CephCluster `spec.dashboard.ssl` is true or false. Since the CephCluster spec is picked up as a single blob from values.yaml, we won't be able to automatically find the value so we will need another variable here to indicate the correct protocol for the port. Or can you think of a better way?

Reply:
Well, I have tried running the chart with `spec.dashboard.ssl: false` and the service port name was still `https`. I wouldn't bother adding logic just for that.

Reply:
Did you at least see the port number change with http vs https? Where the [service is created](https://github.com/rook/rook/blob/81329091da9fb9b01683d472cba4d9179c1d179c/pkg/operator/ceph/cluster/mgr/spec.go#L292-L313), it should be changing the port name as well. 

If we change this to always use https, it doesn't seem like it will work for installs that use http.

Reply:
Well, I just realized the change wasn't instant, it did switch back to `http-`. I need to think more, but maybe a `regexMatch` for `spec.dashboard.ssl` could do the trick?